pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -29,6 +29,7 @@ type PVCToMigrate struct {
 	TargetStorageClass    string                            `json:"targetStorageClass"`
 	TargetAccessModes     []kapi.PersistentVolumeAccessMode `json:"targetAccessModes"`
 	TargetNamespace       string                            `json:"targetNamespace,omitempty"`
+	TargetName            string                            `json:"targetName,omitempty"`
```

Comment:
I would like to see a comment or something here that explains usage, as it appears we have some helper.

It also appears that this is not optional, I believe that we must mark it as such.